src/test/java/com/acrolinx/sidebar/utils/MatchUtilsTest.java

```diff
@@ -25,4 +27,63 @@ void testSortByOffsetDesc() {
     assertEquals("2", sortedMatches.get(2).getContent());
     assertEquals("1", sortedMatches.get(3).getContent());
   }
+
+  @Test
+  void shouldRemoveWhiteSpaceMatch() {
```

Comment:
although JUnit does not require this, we usually add "...Test" as suffix to the method name.

Reply:
Fixed